packages/fxa-auth-server/lib/payments/iap/iap-config.ts

```diff
@@ -2,12 +2,30 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 import { Firestore } from '@google-cloud/firestore';
+import {
+  MozillaSubscriptionTypes,
+  SubscriptionType,
+} from 'fxa-shared/subscriptions/types';
 import { Container } from 'typedi';
 import { TypedCollectionReference } from 'typesafe-node-firestore';
 
 import { AppConfig, AuthFirestore, AuthLogger } from '../../types';
+import { SubscriptionPurchase as AppStoreSubscriptionPurchase } from './apple-app-store/subscription-purchase';
+import { SubscriptionPurchase as PlayStoreSubscriptionPurchase } from './google-play/subscription-purchase';
 import { IapConfig } from './types';
 
+export function getIapPurchaseType(
```

Comment:
I'm not sure where is the best place for this helper, so I stuck it here...

Reply:
Per Ben, this is fine for now just add a comment about why this is here, so if there's a refactor, they know it can be moved.